index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
-    <title>Systers Open Source Projects by Systers</title>
+    <title>HOME- Systers Open Source Projects by Systers</title>
```

Comment:
@Vertika-Shukla again talking about consistence do we need a captialised `HOME`?
Camel Cased `Home` should do, right?